packages/forms-web-app/src/dynamic-forms/dynamic-components/site-address/question.js

```diff
@@ -31,6 +31,22 @@ class SiteAddressQuestion extends Question {
 		this.url = url;
 	}
 
+	prepQuestionForRendering(section, journey, customViewData) {
+		const answers = journey.response.answers.SubmissionAddress[0] || {};
```

Comment:
this should filter the addresses by fieldname before taking the first not take the first otherwise will break if there are any further address questions added in the same journey
there's a util function for this: getAddressesForQuestion